rcl_logging_spdlog/test/test_logging_interface.cpp

```diff
@@ -39,12 +34,40 @@ TEST(logging_interface, init_invalid)
   rcutils_reset_error();
 }
 
-TEST(logging_interface, full_cycle)
+TEST_F(LoggingTest, full_cycle)
 {
-  rcutils_allocator_t allocator = rcutils_get_default_allocator();
+  ASSERT_EQ(0, rcl_logging_external_initialize(nullptr, allocator));
 
+  // Make sure we can call initialize more than once
   ASSERT_EQ(0, rcl_logging_external_initialize(nullptr, allocator));
-  EXPECT_EQ(0, rcl_logging_external_set_logger_level(nullptr, RCUTILS_LOG_SEVERITY_INFO));
-  rcl_logging_external_log(RCUTILS_LOG_SEVERITY_INFO, nullptr, "Log Message");
+
+  std::stringstream expected_log;
+  for (int level = RCUTILS_LOG_SEVERITY_UNSET; level <= RCUTILS_LOG_SEVERITY_FATAL; level += 10) {
```

Comment:
@cottsay I'd rather not make assumptions about enum integer values. Consider using an array instead.

Reply:
I'm not sure this makes me feel any more warm and fuzzy than assuming the int values, but I did it anyway. See 2fcfff9.